plugins/reposync.py

```diff
@@ -71,6 +71,8 @@ def set_argparser(parser):
                             help=_('download only newest packages per-repo'))
         parser.add_argument('-p', '--download-path', default='./',
                             help=_('where to store downloaded repositories'))
+        parser.add_option('--norepopath', default=False, action="store_true",
+                            help=_('Don't add the reponame to the download path.')
```

Comment:
You have syntax error here - either escape: `'Don\'t...` or use double quotes around help string: `"Don't add..."`

Reply:
In fact I had this one escaped but when I copied from the yum version of norepopath it disappeared. Iǘe now used the double quote syntax therefore.